package.json

```diff
@@ -43,7 +44,8 @@
     "sharp": "^0.30.1",
     "starknet": "^6.11.0",
     "ts-node": "^10.8.1",
-    "typescript": "^4.7.3"
+    "typescript": "^5.6.3",
+    "viem": "^2.21.35"
```

Comment:
I don't see this being used, could you remove it?

Reply:
> I don't see this being used, could you remove it?

Our library need these.